### DISCUSSION TITLE
I want you to add Japanese Android and iOS to Lokalise.

### DISCUSSION THREAD

**Discussion by yohru** [home-assistant/frontend#10763] [Localization]
The Japanese translation of the front end and core is almost finished, so I tried to translate the android and iOS, but there was no Lokalise in Japanese.

English Name: Japanese
Native Name: 日本語
Subtag: ja

**Comment by spacegaier** [comment]
Please raise that request in the Android https://github.com/home-assistant/android and iOS https://github.com/home-assistant/iOS repo.

**Comment by dshokouhi** [comment]
I think this has already been completed in both repos
